src/main/java/strategy/FruitStorageHandler.java

```diff
@@ -0,0 +1,37 @@
+package strategy;
+
+import db.FruitStorage;
+import java.util.Map;
+
+public class FruitStorageHandler implements StorageHandler<String,Integer> {
+    private static final String REPORT_HEAD = "fruit,quantity";
+    private static final FruitStorage FRUIT_STORAGE = new FruitStorage();
```

Comment:
Components we rely on are better to be non-static and injected via constructor
```suggestion
    private final FruitStorage storage;
    
    public FruitStorageHandler(FruitStorage storage) {
        this.storage = storage;
    }
```